frontend/server/src/DAO/UserRoles.php

```diff
@@ -162,7 +162,7 @@ public static function getCourseTeachingAssistants(
             $course->acl_id,
         ];
 
-        /** @var list<array{acl: int, username: string}> */
+        /** @var list<array{acl: int, username: string, user_id: int}> */
```

Comment:
Acá parece que al validador de tipos no le gustó el orden, en particular los tipos que se obtienen de consultas de la base de datos deben ir en estricto orden alfabético:
```
        ERROR:root:Mismatched docstring for call in /home/runner/work/omegaup/omegaup/frontend/server/src/DAO/UserRoles.php:167:
	Expected: /** @var list<array{acl: int, user_id: int|null, username: string}> */
	Found:    /** @var list<array{acl: int, username: string, user_id: int}> */
```

```suggestion
        /** @var list<array{acl: int, user_id: int|null, username: string}> */
```

Aunque ahora que lo pienso: ¿realmente necesitamos obtener el `user_id`? Por lo regular los ids no los mandamos a la UI, salvo en contadas excepciones, y me gustaría saber si este es el caso.